runtime/stdlib/test_contract.go

```diff
@@ -106,6 +106,75 @@ var testTypeAssertFunction = interpreter.NewUnmeteredHostFunctionValue(
 	},
 )
 
+// 'Test.assertEqual' function
+
+const testTypeAssertEqualFunctionDocString = `
+Fails the test-case if the given values are not equal, and
+reports a message which explains how the two values differ.
+`
+
+const testTypeAssertEqualFunctionName = "assertEqual"
+
+var testTypeAssertEqualFunctionType = &sema.FunctionType{
+	Parameters: []sema.Parameter{
+		{
+			Label:      sema.ArgumentLabelNotRequired,
+			Identifier: "expected",
+			TypeAnnotation: sema.NewTypeAnnotation(
+				sema.AnyStructType,
+			),
+		},
+		{
+			Label:      sema.ArgumentLabelNotRequired,
+			Identifier: "actual",
+			TypeAnnotation: sema.NewTypeAnnotation(
+				sema.AnyStructType,
```

Comment:
I wonder if equality of two types could be enforced easily.

By that I mean that if types are not equal then the very first test execution is not going to pass anyway. So, its not something that important. On the other hand having comparisons of ints to string fail at the compile time could've been nice too. 

Also, may be there is already a check at the compile time that I've missed.

Reply:
Right, cadence is interpreted language, my bad.

Great, that you've covered that in test. "not equal: expected: true, actual: 1" as discussed in PMs may be error message can specify that types differ.

Reply:
Although, commit is merged and I think this point is "matter of taste"/minor. So, fine for me either way.

Reply:
Great, thanks for pointing it out though :raised_hands: